src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Don't use static methods in your solution. Instead of calling ColorSupplier.getRandomColor() statically, you should create an instance of ColorSupplier and call the method on it.